### PR TITLE
Update DG queryPatient and queryDoctor

### PR DESCRIPTION
**Pull request by Alteqa**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -300,6 +300,33 @@ The sequence diagram below closely describes the interaction between the various
...
+
+* Step 4. The `parse` command in `queryDoctorCommandParser` or `queryDoctorCommandParser` return an instance of `queryPatientCommand` or `queryPatientCommand` respectively.
+* Step 5. The `LogicManager` calls the `execute` method in `queryDoctorCommandParser` or `queryDoctorCommandParser`.
+* Step 6. The `execute` method in `queryDoctorCommandParser` or `queryDoctorCommandParser` executes and calls `updateFilteredPersonList` in model to get a filtered list of `Doctor` or `Patient`.
+* Step 7. Success message gets printed onto the results display to notify user and the list of matching results is produced.
+
+
+Why is this implemented this way?
+1. Making both `Doctor` and `Patient` class extend the `Person` class makes it easier to execute query operations.
+2. `Doctor` and `Patient` all exhibit similar qualities, and thus can inherit from the `Person` superclass.
```
Comment (ararchch) [inline docs/DeveloperGuide.md:327]: can add more implementation detail down the line

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -227,7 +227,7 @@ This command is implemented through the `DeleteCommand` class which extend the `
 
 The activity diagram below demonstrates this error handling process in more detail.
 
-<img src="images/DeletePersonActivityDiagram.png" width="800" />
+<img src="images/QueryPersonActivityDiagram.png" width="800" />
```
Comment (ararchch) [inline docs/DeveloperGuide.md:230]: I think you accidentally edited the wrong activity diagram

**Review by ararchch — CHANGES_REQUESTED** [review]
Some minor changes needed

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Alteqa) [reply docs/DeveloperGuide.md:327]: aight nws

**Review by ararchch — APPROVED** [review]
LGTM

**Pull request merged**